### PR TITLE
Model creation & training

### PR DESCRIPTION
**Pull request by OussamaElhamdani**
Model creation and training: using a feedforward nn (basic architecture of a neural network), 2 hidden layers with a tanh activation and a softmax activation on the output layer, i've trained the model on the data that we have and saved it into chatbot_model.h5 file.

### CODE REVIEW

**Review by s7yby02 — COMMENTED** [review]
Good work
